data/json/furniture_and_terrain/terrain-floors-indoor.json

```diff
@@ -721,6 +721,77 @@
     "flags": [ "TRANSPARENT", "FLAMMABLE_HARD", "SUPPORTS_ROOF", "COLLAPSES", "INDOORS", "FLAT", "RUG", "EASY_DECONSTRUCT" ],
     "deconstruct": { "items": [ { "item": "p_carpet", "charges": 1 }, { "item": "nail", "charges": 5 } ], "ter_set": "t_floor" }
   },
+  {
+    "type": "terrain",
+    "id": "t_carpet_base_wood",
+    "name": "carpet",
+    "description": "Base carpet!",
+    "symbol": ".",
+    "color": "red",
+    "move_cost": 2,
+    "comfort": 1,
+    "roof": "t_flat_roof",
+    "flags": [ "TRANSPARENT", "FLAMMABLE_HARD", "SUPPORTS_ROOF", "COLLAPSES", "INDOORS", "FLAT", "RUG", "EASY_DECONSTRUCT" ],
+    "bash": {
+      "sound": "SMASH!",
+      "ter_set": "t_floor",
+      "str_min": 5,
+      "str_max": 15,
+      "str_min_supported": 100,
+      "items": [ { "item": "cotton_patchwork", "count": [ 10, 20 ] }, { "item": "nail", "charges": [ 1, 3 ] } ]
+    },
+    "deconstruct": { "items": [ { "item": "r_carpet", "count": 1 }, { "item": "nail", "charges": [ 2, 5 ] } ], "ter_set": "t_floor" }
+  },
+  {
+    "type": "terrain",
+    "id": "t_carpet_wood_red",
+    "copy-from": "t_carpet_base_wood",
```

Comment:
it would be nice to set up somme looks like between the carpets to facilitate tileset work

Reply:
Done.